tests/cases/complex.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
```

Comment:
I'd prefer we dont have test cases here when none of the transpiler languages support the test case.
That means there are more failures when running in `UPDATE_EXPECTED=1` mode.

These could be just snippets in `test_unsupported` where they will fail, when they start working, requiring the lang be added to `EXPECTED_SUCCESSES`.

If you dont want to adjust this, I'll figure out some way to have test cases which are known to not work on any target lang, and probably also single out test cases which are also only working on Rust, as most of the Rust only test cases are very unlikely to start working on other targets without a lot of effort by some mythical new contributor.

Reply:
I got julia working.